setup.cfg

```diff
@@ -95,10 +157,10 @@ tracing =
 bentoml = py.typed
 
 [global]
-verbose = true
+quiet = 1
```

Comment:
could you link to related doc for this type of changes?

Reply:
these are global options when you pass to setuptools

Reply:
why are we changing it and do you have link to docs to support this change?

Reply:
this is just to quiet the output from setuptools when developers build bento from wheels file